Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -407,8 +404,11 @@ AppDelegateViewModelOutputs {
       .filter { $0 == .tab(.me) }
       .ignoreValues()
 
-    self.goToMobileSafari = self.foundRedirectUrlProperty.signal.skipNil()
-      .filter { Navigation.deepLinkMatch($0) == nil }
+    self.goToMobileSafari = Signal.merge(
```

Comment:
I *think* you might be able to just use `deeplinkFromUrl.skipNil()` here. Just wanted to make a note of that in case I forget later 😄